cogs/base_cords.py

```diff
@@ -100,11 +100,15 @@ async def createcompanychannels(self, ctx:utils.Context, user:discord.Member, ca
     @utils.command(aliases=['rcs'])
     @commands.has_permissions(manage_channels=True)
     @commands.guild_only()
-    async def removechannelsets(self, ctx:utils.Context, categoryid:int):
+    async def removechannelsets(self, ctx: utils.Context, categoryid: int):
         """Removes channels from a category"""
         async with self.bot.database() as db:
             audit_channel = await db("SELECT audit_channel FROM guild_settings WHERE guild_id=$1", ctx.guild.id)
+            
         cat = self.bot.get_channel(categoryid)
+        if cat is None:
+            return await ctx.send('Invalid category ID!')
```

Comment:
```suggestion
            return await ctx.send('That category could not be found. Make sure you copied the correct ID and that the bot has permission to access it.')
```
maybe this instead? it's more descriptive and provides potential fixes for the problem

Reply:
approved via discord

![image](https://user-images.githubusercontent.com/17860556/123872937-58044d80-d8fb-11eb-8fa6-5d69513404d2.png)